scripts/calculator.js

```diff
@@ -48,14 +48,61 @@ const updateScreen = (screenText = '') => {
 const clickListener = () => {
   updateScreen('1')
 }
-
+const clickListener2 = () => {
+  updateScreen('2')
+}
+const clickListener3 = () => {
+  updateScreen('3')
+}
+const clickListener4 = () => {
+  updateScreen('4')
+}
+const clickListener5 = () => {
+  updateScreen('5')
+}
+const clickListener6 = () => {
+  updateScreen('6')
+}
+const clickListener7 = () => {
+  updateScreen('7')
+}
+const clickListener8 = () => {
+  updateScreen('8')
+}
+const clickListener9 = () => {
+  updateScreen('9')
+}
+const clickListener0 = () => {
+  updateScreen('0')
+}
 // 1.
 const initializeListeners = () => {
   const buttonOne = document.getElementById('button-one')
+  const buttonTwo = document.getElementById('button-two')
+  const buttonThree = document.getElementById('button-three')
+  const buttonFour = document.getElementById('button-four')
+  const buttonFive = document.getElementById('button-five')
+  const buttonSix = document.getElementById('button-six')
+  const buttonSeven = document.getElementById('button-seven')
+  const buttonEight = document.getElementById('button-eight')
+  const buttonNine = document.getElementById('button-nine')
+  const buttonZero = document.getElementById('button-zero')
+  const buttonAllClear = document.getElementById('button-clear')
 
   // listener is attached to "button-one"
   buttonOne.onclick = clickListener
+  buttonTwo.onclick = clickListener2 
+  buttonThree.onclick = clickListener3
+  buttonFour.onclick = clickListener4
+  buttonFive.onclick = clickListener5
+  buttonSix.onclick = clickListener6
+  buttonSeven.onclick = clickListener7
+  buttonEight.onclick = clickListener8 
+  buttonNine.onclick = clickListener9
+  buttonZero.onclick = clickListener0
+  buttonAllClear.onclick = updateScreen('')
```

Comment:
On why this isn't working:

Compare how "updateScreen" and "clickListener0" are being used here.
On a more basic level, consider this example:
```
var someFunction = () => {
  return 'potato salad'
}

var x = someFunction()
var y = someFunction

console.log(x)
console.log(y)
```
What will x be? What will y be?

The onclick 'listener' is a _function_ we specify to be run when a click 'event' occurs. The problem on line 103 is that we aren't specifying a function: we're specifying whatever the result of calling "updateScreen" is. Since "updateScreen" doesn't return anything, onclick is set to "undefined".